pkg/analyze/front/front.tmpl

```diff
@@ -0,0 +1,55 @@
+{{ define "resources" -}}
+	Resources GVK: {{- "\n" -}}
+	{{- range $key, $value := . -}}
+		{{- if $value -}}
+        	{{- "\t" -}}
+			{{- if $value.GetObjectKind.GroupVersionKind.Empty -}}
+				No GVK set {{- "\n" -}}
+			{{- else -}}
+				{{- $gvk := $value.GetObjectKind.GroupVersionKind -}}
+				{{- $gvk.Group }}/{{ $gvk.Version }}.{{ $gvk.Kind }}, {{ $value.Namespace -}}
+				{{- if and $value.Namespace $value.Name -}}
+					/
+				{{- end -}}
+				{{- $value.Name -}}
+				{{- if $value.GetUID -}}
+					{{- " " -}} ({{ $value.GetUID }})
+				{{- end -}}
+				{{- "\n" -}}
+			{{- end -}}
+		{{- end -}}
+	{{- end -}}
+{{end}}
+
+{{ define "symptom" -}}
+	{{.Name}} {{- "\n" -}}
```

Comment:
nit: not sure about printing the name, it seems like an internal detail again, but I'm leaving this up to you